### PR TITLE
Issue 56 robotframework event create test

### PR DESCRIPTION
**Pull request by vladislav-kulakovsky**

### CODE REVIEW

**Comment by vladislav-kulakovsky** [comment]
I have thought that I will do issue-57 in this branch

**Comment by vladislav-kulakovsky** [comment]
I have thought that I will do issue-57 in this branch

File: src/test/robot/api/storage/create_events.robot [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    Remove Directory  ${LOG_DIR}/${step_id}  recursive=True
+    ${args} =  Catenate  SEPARATOR= \\\n\t
+    ...  --load-step-id=${step_id}
+    ...  --storage-net-node-port=${PRAVEGA_CONTROLLER_PORT}
+    ...  --load-op-limit-count=10
+    ...  --storage-driver-limit-concurrency=1
+    ${std_out} =  Execute Mongoose Node  ${args}
+    Log  ${std_out}
+    Validate Metrics Total Log File  ${step_id}  CREATE  10  0  10485760
+
```
Comment (akurilov) [inline src/test/robot/api/storage/create_events.robot:35]: extra empty line

File: src/test/robot/api/storage/create_events.robot [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+*** Keyword ***
+Execute Mongoose Node
```
Comment (akurilov) [inline src/test/robot/api/storage/create_events.robot]: Let's name it "Execute Mongoose Scenario"

File: src/test/robot/api/__init__.robot [hunk 3]
```diff
@@ -0,0 +1,3 @@
+*** Settings ***
+Documentation  Mongoose API suite
+Force Tags  API
```
Comment (akurilov) [inline src/test/robot/api/__init__.robot]: missing newline

File: src/test/robot/api/storage/__init__.robot [hunk 4]
```diff
@@ -0,0 +1,3 @@
+*** Settings ***
+Documentation  Mongoose Storage API suite
+Force Tags  Storage API
```
Comment (akurilov) [inline src/test/robot/api/storage/__init__.robot]: missing newline

**Review by akurilov — APPROVED** [review]

**Pull request merged**